common/test/acceptance/pages/studio/course_page.py

```diff
@@ -17,6 +18,15 @@ class CoursePage(PageObject):
     # Does not need to include the leading forward or trailing slash
     url_path = ""
 
+    @abstractmethod
+    def is_browser_on_page(self):
+        """
+        Verifies browser is on the correct page.
+
+        Should be implemented in child classes.
+        """
+        pass
```

Comment:
@Muddasser Why we need this `abstractmethod`? As per my understanding the correct way to declare an abstractmethod is that either you should mark the class as abstract by declaring `__metaclass__ = abc.ABCMeta` in `CoursePage` or raise `raise NotImplementedError()` exception in `is_browser_on_page`?